api/types/build/disk_usage.go

```diff
@@ -1,6 +1,8 @@
 package build
 
 // CacheDiskUsage contains disk usage for the build cache.
+//
+// Deprecated: this struct has been deprecated and will be removed in the next release.
```

Comment:
Similar to the other PR; probably better to make this a separate PR for the 28.x branch that _only_ adds the deprecation message, then in this PR make it a commit that moves it (no deprecation step between).